expressEngine/login/utils/login.js

```diff
@@ -0,0 +1,33 @@
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+
+class UserHelper {
+  getProjection = () => {
+    const projection = {
+      email: 1,
+      password: 1,
+    };
+    return projection;
+  };
+
+  checkUser = async ({ loginPassword = '', hashPassword = '' }) => {
+    try {
+      const match = await bcrypt.compare(loginPassword, hashPassword);
+      return match;
+    } catch (err) {
+      return { error: err };
+    }
+  };
+
+  createToken = async ({ data }) => {
+    try {
+      const { key = 'successive' } = process.env;
```

Comment:
the key is in all caps const { KEY = 'successive' }

Reply:
key also, work without caps

Reply:
it works because of every time it takes default value not env key, if you remove default value it will not work